test/vms.full.test.js

```diff
@@ -903,7 +904,16 @@ exports.create_vm = function (t) {
         locality: {
             strict: false,
             far: [ '00000000-0000-0000-0000-000000000001' ]
-        }
+        },
+        affinity: [
+            {
+                "key": "instance",
+                "operator": "!=",
+                "value": "00000000-0000-0000-0000-000000000001",
```

Comment:
Not a blocker but I wish to understand:  This rule means "DO NOT MIGRATE ME TO CN UUID == 000....00000001" right?

Reply:
It means "do not provision to vm with uuid ...", and line 913 shows that it's a soft requirement, so if the constraint can't be satisfied the provision is still allowed anyway.